Flow Actions/Assign Role/assignRole.js

```diff
@@ -0,0 +1,20 @@
+(function execute(inputs, outputs) {
+
+    var user = inputs.user_id;
+    var roleSysId = inputs.role_sys_id;
```

Comment:
I also don't see the value in using the sys_id of the role. It would be much better to just take the role name, such as `user_admin`.

Reply:
Hi Lacah, As to assign a role to any user we need to create a record in "sys_user_has_role" table for that user. This table contains a role field, and user field. As both fields are reference fields, so we need to use the sys_id of the role.